understanding/seizures-and-physical-reactions.html

```diff
@@ -31,7 +31,7 @@ <h1>ガイドライン 2.3: 発作と身体的反応を理解する</h1>
       <main>
          <section xmlns:wcag="https://www.w3.org/WAI/GL/" id="intent">
             <h2>意図</h2>
-            <p>発作性障害のある利用者は、閃光を放つ視覚的なコンテンツによって発作を引き起こすことがある。ほとんどの利用者は、発作が起こすまでは自分がこの疾患を持っていることに気づかない。1997 年に、日本でテレビのアニメ番組が 700 人以上の子どもたちを病院に搬送させる事態を招き、そのうち約 500 人が発作を引き起こした。特に、警告を実際に読むことができない子どもの利用者に警告はよく見逃されるので、その警告はあまり効果がない。</p>
+            <p>発作性障害のある利用者は、閃光を放つ視覚的なコンテンツによって発作を引き起こすことがある。ほとんどの利用者は、発作が襲うまでは自分がこの障害を持っていることに気づかない。1997 年に、日本でテレビのアニメ番組が 700 人以上の子どもたちを病院に搬送させる事態を招き、そのうち約 500 人が発作を引き起こした。特に、警告を実際に読むことができない子どもの利用者に警告はよく見逃されるので、その警告はあまり効果がない。</p>
```

Comment:
これでも伝わると思うのですが、「発作に襲われる」というほうがより自然ではないでしょうか (好みの問題かもしれませんが……)。

```suggestion
            <p>発作性障害のある利用者は、閃光を放つ視覚的なコンテンツによって発作を引き起こすことがある。ほとんどの利用者は、発作に襲われるまでは自分がこの障害を持っていることに気づかない。1997 年に、日本でテレビのアニメ番組が 700 人以上の子どもたちを病院に搬送させる事態を招き、そのうち約 500 人が発作を引き起こした。特に、警告を実際に読むことができない子どもの利用者に警告はよく見逃されるので、その警告はあまり効果がない。</p>
```